lib/aytests/image_builder.rb

```diff
@@ -164,6 +165,13 @@ def import
     def export_from_veewee
       Dir.chdir(work_dir) do
         log.info "Exporting #{veewee_provider} image into box file"
+        # veewee changes the owner of this file to root.
+        # So the call "qemu-img convert" has no access anymore.
+        # --> Access has to be set manually.
+        if File.file? CLONE_IMAGE_PATH
+          log.info "Giving write permissions to #{CLONE_IMAGE_PATH}"
+          system "chmod 666 #{CLONE_IMAGE_PATH}"
```

Comment:
I would use [FileUtils.chmod](http://ruby-doc.org/stdlib-2.2.0/libdoc/fileutils/rdoc/FileUtils.html#method-c-chmod) to perform this operation (instead of relying in `system`).